examples/alignment/tracking/tracking_kf_alignment.lcsim

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      Steering file for running KF tracking and mille alignment
+      created:  Nov 2022
+      @author Tom Eichlersmith <eichl008@umn.edu>
+      @author PF <pbutti@slac.stanford.edu>
+    -->
+    <execute>
+      
+      <!-- Enable the following if re-processing lcio files -->
+      <driver name="PreCleanupDriver"/>      
+      
+      <driver name="RfFitter"/>
+      <driver name="EcalRunningPedestal"/> 
+      <driver name="EcalRawConverter" />
+      <driver name="EcalTimeCorrection"/> 
+      <driver name="ReconClusterer" /> 
+      <driver name="CopyCluster" /> 
+      
+      <!-- 
+        SVT reconstruction drivers 
+    
+        SensorSetup associates hits with sensors without having to re-decode them.
+        TrackerHitDriver assocites clusters with tracker modules without having to re-cluster.
+      -->
+      <driver name="SensorSetup"/>
+      <driver name="TrackerHitDriver"/> 
+
+      <!--
+        Kalman tracking
+      -->
+      <driver name="KalmanPatRecDriver"/>
+      <driver name="KalmanKinkFitDriver"/>
+      <driver name="KalmanToGBLDriver"/> 
```

Comment:
Could this be interfering with the `GBLTracks` that end up being used for calculating the residuals/derivatives?

Reply:
Nope, it makes no difference. I ran with and without this driver and both produced identical mille files.